src/routes/(app)/maluspunten/+page.svelte

```diff
@@ -1,79 +1,84 @@
-  import { page } from '$app/stores'
-  import { enhance, applyAction } from '$app/forms'
-	import StyledButton from '$lib/components/StyledButton.svelte';
+  import { page } from "$app/stores";
+  import { enhance, applyAction } from "$app/forms";
+  import StyledButton from "$lib/components/StyledButton.svelte";
 
-  let error = ''
+  let error = "";
 
   function formatDate(dateString: string) {
-    const date = new Date(dateString)
+    const date = new Date(dateString);
 
-    const hour = ('0' + date.getHours()).slice(-2)
-    const minute = ('0' + date.getMinutes()).slice(-2)
+    const hour = ("0" + date.getHours()).slice(-2);
+    const minute = ("0" + date.getMinutes()).slice(-2);
 
-    return `${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()} ${hour}:${minute}`
+    return `${date.getDate()}/${
+      date.getMonth() + 1
+    }/${date.getFullYear()} ${hour}:${minute}`;
   }
 </script>
 
-<div>
+<main>
   <h1>Maluspunten</h1>
 
   <hr />
 
-  <div id="form-container">
-    <form method="POST" use:enhance={event => {
-      return async ({ result, update }) => {
-        console.log(result)
-        if (result.type === 'failure') {
-          error = result.data.message
-        } else {
-          error = ''
-          console.log(result)
-          update()
-        }
-      }
-    }}>
+  <form-container>
+    <form
+      method="POST"
+      use:enhance={(event) => {
+        return async ({ result, update }) => {
+          console.log(result);
+          if (result.type === "failure") {
+            error = result.data?.message;
+          } else {
+            error = "";
+            console.log(result);
+            update();
+          }
+        };
+      }}
+    >
       <div>
         <label for="receiverId">Welke feut</label>
         <select name="receiverId">
           {#each $page.data.feuten as user}
-            <option value="{user.id}">{user.firstName}</option>
+            <option value={user.id}>{user.firstName}</option>
           {/each}
         </select>
       </div>
-  
-      <div>  
+
+      <div>
         <label for="reason">Waarom</label>
         <input type="text" name="reason" placeholder="Waarom" />
       </div>
-  
-      <div>  
+
+      <div>
         <label for="amount">Hoeveel maluspunten</label>
         <input type="number" name="amount" placeholder="Hoeveel" />
       </div>
-      
+
       <div>
         <label for="giverId">Gegeven door</label>
         <select name="giverId">
           {#each $page.data.members as user}
-            <option value="{user.id}">{user.firstName}</option>
+            <option value={user.id}>{user.firstName}</option>
           {/each}
         </select>
       </div>
-      
-      <div>  
+
+      <div>
         <StyledButton type="submit">Toevoegen</StyledButton>
       </div>
-      
+
       <div>
         {error}
       </div>
     </form>
-  </div>
+  </form-container>
 
   <hr />
 
-  <div id="table-container">
+  <table-container>
```

Comment:
huh

Reply:
dit is cooler maar geen idee meer precies wat ik daar allemaal me heb zitten kutten